website_sale_one_step_checkout_delivery/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "One Step Checkout Delivery",
+    'category': 'Website',
+    'summary': """Add Delivery Costs to One-Step-Checkout""",
+    'description': """
+Delivery Costs
+==============
```

Comment:
Add here the fact that delivery methods can be selected in the one step checkout and that delivery changes are done without page reloading now.